pkg/solidity-utils/test/ERC20Permit.test.ts

```diff
@@ -86,7 +87,14 @@ describe('ERC20Permit', () => {
           const currentNonce = await token.nonces(holder.address);
           const otherToken = await deploy('ERC20PermitMock', { args: ['Token', 'TKN'] });
 
-          ({ v, r, s, deadline } = await signPermit(otherToken, holder, spender, amount, currentNonce));
+          ({ v, r, s, deadline } = await signPermit(
+            otherToken,
+            holder,
+            spender.address,
+            amount,
+            MAX_DEADLINE,
```

Comment:
Can't we use the implicit deadline and nonce?

Reply:
The same question applies to usage of the other `sign` methods

Reply:
In this case yes, we shouldn't enforce their use though.